packages/forms/src/widgets/MultiSelectTagWidget/MultiSelectTagWidget.js

```diff
@@ -156,18 +156,18 @@ class MultiSelectTagWidget extends React.Component {
 		}
 		case keycode.codes.up: {
 			if (this.state.selectedIndex > 0) {
-				this.setState({
-					selectedIndex: this.state.selectedIndex - 1,
-				});
+				this.setState(prevState => ({
+					selectedIndex: prevState.selectedIndex -1,
+				}));
```

Comment:
i don't see the point of doing that here since what the function is doing here is really simple.